### PR TITLE
Amendments to the reconsent.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

https://www.notion.so/joinzoe/Reconsent-v2-0-KCL-Amendments-d7ae108e313d40f99742b7d5ed178d0d

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

New option `Prefer not to say`:
Simulator Screen Shot - iPhone 11 - 2021-08-17 at 12 34 51

Changed title:
Simulator Screen Shot - iPhone 11 - 2021-08-17 at 12 34 55

Changed subtitle:
Simulator Screen Shot - iPhone 11 - 2021-08-17 at 12 34 58

Link to information sheet:
Simulator Screen Shot - iPhone 11 - 2021-08-17 at 12 35 01

Reorder of the options:
Simulator Screen Shot - iPhone 11 - 2021-08-17 at 12 35 05

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

### CODE REVIEW

**Review by bionara — APPROVED** [review]

File: src/core/state/reconsent/types/index.ts [hunk 1]
```diff
@@ -10,15 +10,16 @@ export type TDiseaseId =
...
+  | 'prefer_not_to_say';
 
 export type TDiseasePreferencesData = {
   [key in TDiseaseId]?: boolean;
 };
 
 export type TFeedbackId =
-  | 'im_only_interested_in_fighting_covid_19_right_now'
-  | 'commercial_purposes'
+  | 'i_only_want_to_log_my_symptoms'
```
Comment (selvach) [inline src/core/state/reconsent/types/index.ts:21]: I don't plan to change the backend fields. You can revert the id to their original values and just update the ocopy!

File: src/features/reconsent/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import * as React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+
+type TProps = {
+  description: string;
+  index: number;
+  title: string;
+};
+
+export function Card(props: TProps) {
```
Comment (selvach) [inline src/features/reconsent/components/Card.tsx:13]: Refactor King 👑

**Review by selvach — CHANGES_REQUESTED** [review]
Please revert the ID's as we won't change the backend for this.

File: assets/lang/en.json [hunk 3]
```diff
@@ -1453,7 +1453,7 @@
   "reconsent": {
     "disease-summary": {
       "button": "Next",
-      "title": "We’ll invite you to answer additional questions occasionally and send you insights about",
+      "title": "Beyond your daily reporting, we'll ask extra questions and present insights about",
```
Comment (bionara) [inline assets/lang/en.json:1456]: note - could prob just use a new line vs having two json keys

**Review by bionara — APPROVED** [review]

**Review by selvach — APPROVED** [review]

**Pull request merged**